### PR TITLE
new icon: azuredevops (original, plain)

### LINKED ISSUE #1281: [ICON REQUEST]: Azure DevOps
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Services for teams to share code, track work, and ship software (Microsoft's answer to GitHub, before they acquired GitHub)

### Links and sources

[Website](https://azure.microsoft.com/en-us/services/devops/)

SVG (From their website)
Logo

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1281**

## Link to prove your SVG is correct and up-to-date.

See #1281.

### CODE REVIEW

**Review by Snailedlt — CHANGES_REQUESTED** [review]
The folder, icons and PR title should be renamed to `azuredevops`. See the [Naming-Conventions](https://github.com/devicons/devicon/wiki/Naming-Conventions) wiki for more info

File: devicon.json [hunk 1]
```diff
@@ -1017,6 +1017,26 @@
...
         "aliases": []
     },
+    {
+        "name": "azuredevops",
+        "altnames": [
+            "azure-devops"
+        ],
+        "tags": [
+            "azure",
+            "devops"
```
Comment (Snailedlt) [inline devicon.json:1027]:
```suggestion
            "devops",
            "cloud",
            "version control",
            "vcs"
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
There's also some conflicts that need to be solved before we can merge this

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy_check_mark:

**Comment by BenSouchet** [comment]
@kilian-paquier
Please, make sure, when you add a new entry in the `devicon.json` file that you add it at the correct position in the file.
Elements order need to reflect the `icons/` folder, in this PR you added `azuredevops` between `clojurescript` and `cloudflare`.

It hard to keep track of everything and this can lead to mistake when new elements are not added at the correct alphabetic position.

I think this isn't the first time you added a new element to `devicon.json` at "random" possition, I have seen `lumen` placed between `ceylon` and `chrome`.

Because of these misplacements I had an had time fixing merge conflicts in my own PRs.

So if you can ensure for your future contributions that you place new entries at the right alphabetic place in the `devicon.json` file, thanks.

@Snailedlt Maybe It will be another element to add to the checklist when reviewers look at PRs, and maybe a good element to add to documentation for new contributors.

Have a good day guys ☀️

**Comment by Snailedlt** [comment]
@BenSouchet Ohh yeah, didn't notice that. My bad for merging.

On the note of adding this to the reviewing process, I agree. I think we can even do one better, and automatically sort the devicon.js file with some type of linter. Should be possible to do before merge, so we avoid merge conflicts in the future.

I'll make an issue on this once I get back home, so we can discuss it further :)

Thanks for letting me know

**Comment by BenSouchet** [comment]
@Snailedlt I already created this Feature Request #1327 , so everyone can comment / add elements or feedback on the idea.

And yes, automatically sort the JSON file would be great since some entries are already misplaced.
So we could iterate on a proper ordered file for future contributions, and avoid continuing to misplace entries.

**Pull request merged**